content/hacking-atom/sections/package-modifying-text.md

```diff
@@ -22,26 +22,31 @@ The final package can be viewed at https://github.com/atom/ascii-art.
 
```

Comment:
Line 19 above says:

>  The final package can be viewed at https://github.com/atom/ascii-art.

@ungb: Can I talk you into opening a PR to convert https://github.com/atom/ascii-art from CoffeeScript to JavaScript? I think it would be helpful to get that PR ready to go first, and then copy the resulting JavaScript code into this PR. That way, we can handle all the niggling CoffeeScript-to-JavaScript conversion issues (e.g., removing [unnecessary return statements](https://github.com/atom/flight-manual.atom.io/blame/51116540ffff47d45dfb84c95f86bb88acd1a228/content/hacking-atom/sections/package-modifying-text.md#L47), adopting JavaScript Standard Style, etc.) in that PR, and this PR can mostly be an exercise in copying the new JavaScript code from https://github.com/atom/ascii-art into the right places in this tutorial.

What do you think?

Reply:
👍